src/app/legacy/components/Footer/Link/index.jsx

```diff
@@ -17,22 +17,55 @@ const StyledLink = styled.a`
   }
 `;
 
-const Link = ({ service, text, href, inline, lang }) => (
-  <StyledLink
-    service={service}
-    lang={lang}
-    inline={inline}
-    href={href}
-    className="focusIndicatorInvert"
-  >
-    {text}
-  </StyledLink>
-);
+const Link = ({
+  service,
+  text,
+  href,
+  inline,
+  lang,
+  onClick,
+  onlyShowIfJSenabled,
+  visible,
+}) => {
+  const [isVisible, setVisible] = useState(onlyShowIfJSenabled !== true);
+  const [clientside, setClientside] = useState(false);
+
+  useEffect(() => {
+    if (onlyShowIfJSenabled) {
+      if (clientside) {
+        setVisible(visible);
+      } else {
+        setClientside(true);
+      }
+    }
+  }, [onlyShowIfJSenabled, clientside, visible]);
```

Comment:
We have a utility called `onClient()` in `#lib/utilities/onClient` which returns `true | false` if it detects that the request is client or server side. May be useful as this looks like its checking something similar.

Reply:
+1